SU-HSDO/stanford_mrc
# MRCD8-308 main menu fix

# READY FOR REVIEW

# Summary
- Added padding to header region to match comps more closely
- Addressed closeness of underline on menu items

# Needed By (Date)
- End of sprint (3/29)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the spacing on the header looks good
3. Ensure that there is an underline (bottom border) on menu items with some space

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-308

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
